testsuite/scripts/gha_test_run_proto.json

```diff
@@ -0,0 +1,188 @@
+    {
+      "test":"*",
+      "suite":"TestFilterUGens",
+      "skip":true,
+      "skipReason":"UnitTest has bugs (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestLinkClock",
+      "skip":true,
+      "skipReason":"UnitTest has unknown issues when run in qpm (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestMixedBundleTester",
+      "skip":true,
+      "skipReason":"UnitTest has bugs (FIXME)"
+    },
+    {
+      "test":"test_copy_nodeMapIsCopied",
+      "suite":"TestNodeProxy",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"test_audiorate_mapping",
+      "suite":"TestNodeProxyBusMapping",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"test_audiorate_mapping_elasticity",
+      "suite":"TestNodeProxyBusMapping",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"test_proxyroles_xfade_smoothly",
+      "suite":"TestNodeProxyRoles",
+      "skip":true,
+      "skipReason":"UnitTest fails in qpm (FIXME)"
+    },
+    {
+      "test":"test_synthDef_isReleased_afterFree",
+      "suite":"TestNodeProxy_Server",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"test_schedAfterFade_notBeforeQuant",
+      "suite":"TestNodeProxy_Server",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestNode_Server",
+      "skip":true,
+      "skipReason":"UnitTest has bugs (FIXME)"
+    },
+    {
+      "test":"test_argSizeMatching_Succeeds",
+      "suite":"TestOSCFunc",
+      "skip":true,
+      "skipReason":"UnitTest fails when run in qpm on Linux (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestPanAz",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestProxyNodeMap",
+      "skip":true,
+      "skipReason":"UnitTest has bugs (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestPV_ChainUGen",
+      "skip":true,
+      "skipReason":"Server might hang after these tests when run in qpm (FIXME)"
+    },
+    {
+      "test":"test_helpSourceDirs_includedExtensions",
+      "suite":"TestSCDoc",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestServer_boot",
+      "skip":true,
+      "skipReason":"UnitTest has bugs (FIXME)"
+    },
+    {
+      "test":"test_nextTimeOnGrid_okAfterMeterChange",
+      "suite":"TestTempoClock",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"test_setUpClass_already_happened",
+      "suite":"TestUnitTest",
+      "skip":true,
+      "skipReason":"UnitTest fails (FIXME)"
+    },
+    {
+      "test":"*",
+      "suite":"TestWebView",
+      "skip":true,
+      "skipReason":"UnitTest sometimes fails when run in qpm on Linux (FIXME)"
+    }
+  ]
+}
```

Comment:
please add a final newline and indent this file at 4 spaces. if you don't mind, i'd prefer seeing "suite" lines before "test" since that feels more intuitive (general to specific)

Reply:
Done! Re: "test" vs "suite" - I was just following the [convention in the previous file](https://github.com/supercollider/supercollider/blob/develop/travis_test_run_proto.json), but I like the idea of changing it into "general to specific"